packages/web/components/Landing/sections/Cards.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, HStack, Image, ListItem, OrderedList, Text } from '@metafam/ds';
+import CardBackground from 'assets/landing/card-background.png';
+import CardImage from 'assets/landing/card-image.png';
+
+function Intro() {
+  return (
+    <Box
+      width="100%"
+      minHeight="1040px"
```

Comment:
Please replace all occurrences of px with equivalent in rem or chakra size defaults such as md, lg or xl.